skforecast/ForecasterRnn/utils.py

```diff
@@ -0,0 +1,125 @@
+################################################################################
+#                             skforecast.utils                                 #
+#                                                                              #
+# This work by Joaquin Amat Rodrigo and Javier Escobar Ortiz is licensed       #
+# under the BSD 3-Clause License.                                              #
+################################################################################
+# coding=utf-8
+
+from typing import Union, Any, Optional, Tuple, Callable
+import pandas as pd
+
+import tensorflow as tf
+from tensorflow import keras
```

Comment:
We need to add tensorflow to the optional dependences

Reply:
Done. See [commit](https://github.com/JoaquinAmatRodrigo/skforecast/pull/581/commits/a3a075eb0896f0bc9606145860bfc1407d3e8afb)

Reply:
@JoaquinAmatRodrigo please, review